### ISSUE TITLE
when use .eslintrc.js, the object 'key' must have quotes

### ISSUE THREAD

**Issue by catshub** [ghmcadams/vscode-lintlens#2]
when i use the .eslintrc.js, the export object's `'rule'` property and the keys of  rules in the `'rule'` property must use `quotes`.

```js
// extension works
module.exports = {
  'rules': {
    'somerule': 1
  }
}
```
```js
// extension will not work
module.exports = {
  rules: {
    'somerule': 1
  }
}
// or
module.exports = {
  'rules': {
    somerule: 1
  }
}
```

vscode: 1.25.1
os: macOS high sierra 10.13.6
LintLens: 1.1.0

**Comment by rodpl** [comment]
This is resolved in #1 . Please be patient and wait for release

**Comment by rodpl** [comment]
Update ... resolved in #3

**Comment by ghmcadams** [comment]
partially resolved in [#3].  published a new version (v2.0.2) which fully resolves this issue.
